src/stories/Autocomplete/Hook/index.tsx

```diff
@@ -119,32 +119,41 @@ export function HooksTemplate(args) {
             if (!section?.data?.length) {
               return null;
             }
+
             const { type, displayName } = section;
-            let sectionName = section.displayName;
+            const sectionListingType = isCustomSection(section)
+              ? 'custom'
+              : toKebabCase(section.indexSectionName || section.data[0]?.section || 'Products');
 
+            let sectionTitle: string;
```

Comment:
Same for sectionTitle. Is there a way we can get this from getSectionProps instead?